src/isar_turtlebot/utilities/pose_message.py

```diff
@@ -1,3 +1,5 @@
+from typing import Dict
```

Comment:
Is this new import used anywhere?

Reply:
Yes, and it was used before as well although there was no import. Weird that there were no errors during runtime. I guess that's just the way Python deals with typehinting :D

Reply:
Although `dict` would have been fine too